### DISCUSSION TITLE
Yet another Hinode themed docs site

### DISCUSSION THREAD

**Discussion by vinser** [gethinode/hinode#1136] [Show and tell]
Tired of maintaining several long README in several languages, I decided to make a separate docs site with a publication on GitHub. Since my program is written in Golang, I used HuGo and not Jekyll ;).
 
The native Hugo Doxy theme allowed me to quickly make a multilingual site with documentation, but I wanted some more. And in the end - a site with a Hinode theme published on GitHub - https://vinser.github.io/flibgolite-docs 

Since I am not an expert in website layout, I can’t say that it was easy as ABC for me to do this. Nevertheless, having dug into the guts of this theme, I got what I wanted finally. 

Some minor issues were related to localization, since due to the site being published on GitHub, baseURL has a repository subdomain. And @markdumay - the theme developer have made the necessary edits to the code almost immediately and everything worked out.

Building on this positive experience, I intend to create several more Hinoded sites to document my other projects.

Thanks to Mark for his really High level theme and lighting fast support!

**Comment by markdumay** [comment]
Thanks for sharing @vinser! Glad to help - and pleased we were able to get the subfolder deployment working.
